IAMOTZ/More-recipes
# #152322339 Set up Server

#### What does this PR do?
Set up node + express server with a dummy database.
#### Description of Task to be completed?
Set up the server side of the application 
#### How should this be manually tested?
- clone the repo
- cd into the server directory
- Run the following command:
`npm install` to install all the dependency packages
`npm run start:dev` to start the server on port 3000
- Visit `localhost:3000/api` in the browser or postman or any other API testing tool you should see:
`{status: 'welcome to the More-recipe api'}`
#### Any background context you want to provide?
I have created all the files for the controllers in this branch,
the branches for the API endpoints would just edit those files.
#### What are the relevant pivotal tracker stories?
#152322339
#### Screenshots (if appropriate)
None
#### Questions:
None
